test/unit/role_test.rb

```diff
@@ -133,6 +133,16 @@ class RoleTest < ActiveSupport::TestCase
       it { subject.must_include(first) }
       it { subject.wont_include(second) }
     end
+
+    context "when current user is admin for_current_user should return all roles" do
+      before do
+        User.current = users(:admin)
+      end
+      tmp_role = Role.create(:name => 'First')
```

Comment:
@alongoldboim What is this `tmp_role` for? It's not being tested anywhere, if you remove it, the test will still pass.

Maybe you want to test it like:

```ruby
Role.expects(:where).with('0 = 0')
```

to ensure it tests the change you made, no need to actually check what's inside the db call. Does that sound correct?